app/views/spree/frontend/products/_subscription_line_item_fields.html.erb

```diff
@@ -0,0 +1,30 @@
+<h3>Subscription attributes</h3>
+<%= fields_for :'subscription_line_item', SolidusSubscriptions::LineItem.new do |ff| %>
+  <div>
+    <%= ff.label :quantity, "I want" %>
+    <%= ff.number_field :quantity %>
+    <%= ff.label :quantity, "items" %>
+  </div>
+
+  <div>
+    <%= ff.label :interval_length, "every" %>
+    <%= ff.number_field :interval_length %>
+
+    <%= ff.collection_radio_buttons :interval_units, SolidusSubscriptions::LineItem.interval_units.to_a, :first, :first %>
+  </div>
+
+  <%= ff.hidden_field :subscribable_id %>
+<% end %>
```

Comment:
@tvdeyen absolutely thanks for catching this :) 

I really only intend this to be used as a demo of how the subscriptions gem can work. But I agree, it takes no time for me to toss it into a translation and its definitely a better change